fleet/lib/metallb-conf/overlays/yagan/ipaddresspool-htcondor.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: htcondor
+  namespace: metallb-system
+spec:
+  addresses:
+  - 139.229.160.221/32
```

Comment:
Side comment: It looks like some of our `IPAddressPool` and `L2Advertisement` and indenting yaml lists and some aren't. We should normalize that.

Reply:
I can fix that on another PR